packages/office-ui-fabric-react/src/components/DocumentCard/DocumentCardTitle.tsx

```diff
@@ -137,14 +134,12 @@ export class DocumentCardTitle extends BaseComponent<IDocumentCardTitleProps, ID
       const { truncatedTitleFirstPiece, truncatedTitleSecondPiece } = this.state;
       this._isTruncated = true;
 
-      if (!truncatedTitleFirstPiece && !truncatedTitleSecondPiece) {
-        this._startTruncation(this.props);
```

Comment:
Where did this line go? Is it not needed anymore?

Reply:
@Markionium yes, I believe it's an extra unnecessary check. `_shrinkTitle()` is called only when the component is already mounted and `_startTruncation()` was called once in `componentDidMount()` where also is attached a `resize` listener which in turn calls `_startTruncation()` again. Next `componentDidUpdate()` is fired and reaches the `_shrinkTitle()`. Initially, the block I removed is skipped as we should have both segments already defined and we are slicing them little by little. The issue comes in play when you resize the window small enough so that the slicing eventually sets them to empty strings and now the block I removed is executed calling `_startTruncation()` which will `setState` causing the `componentDidUpdate()` to call `_shrinkTitle()` and comes to the same check over and over again because the strings will remain empty thus causing the infinite loop.

I hope this explains why it was removed.

Reply:
I believe this strings could reach empty to this method only if you would refresh the shrinked window and after the mount you start the truncation and set them to`''` in `_startTruncation()`. So why call `_startTruncation()` again over and over if the result will not change. Instead I put a check on the slicing below this code block.

Reply:
Sounds sensible to me. I just wanted to make sure it wasn't accidentally removed :D